### PR TITLE
Update Kotlin 1.7.10

### PR DESCRIPTION
**Pull request by Goooler**

### CODE REVIEW

**Comment by Goooler** [comment]
Seems can't resolve `kotlinext` package.

File: buildSrc/src/main/kotlin/dependencies.gradle.kts [hunk 1]
```diff
@@ -207,6 +207,7 @@ val dependencies: List<OpenSourceLibrary> = listOf(
 
     OpenSourceLibrary("org.jetbrains.kotlin-wrappers:kotlin-react:$kotlinReactVersion"),
     OpenSourceLibrary("org.jetbrains.kotlin-wrappers:kotlin-react-dom:$kotlinReactVersion"),
+    OpenSourceLibrary("org.jetbrains.kotlin-wrappers:kotlin-extensions:1.0.1-pre.382"),
```
Comment (Goooler) [inline buildSrc/src/main/kotlin/dependencies.gradle.kts:210]: https://github.com/JetBrains/kotlin-wrappers/tree/master/kotlin-extensions

File: .github/workflows/build-test-webeditor.yml [hunk 2]
```diff
@@ -76,11 +76,10 @@ jobs:
         working-directory: github1s
       - run: yarn serve > ./output 2>&1 &
         working-directory: github1s
-      - name: Set up JDK 11
-        uses: actions/setup-java@v3
+      - uses: actions/setup-java@v3
         with:
           distribution: 'zulu'
-          java-version: 11
+          java-version: 17
```
Comment (blindpirate) [inline .github/workflows/build-test-webeditor.yml:82]: Why 17? I suppose there would be some issues with 17. Any compelling reason to upgrade to 17?

**Review by blindpirate — COMMENTED** [review]
Thanks for the PR, I've planned the upgrade for a long time. I'm a bit busy right now and I'll take a look at it on next Tuesday.

File: .github/workflows/build-test-webeditor.yml (hunk 2 above)
Comment (Goooler) [reply .github/workflows/build-test-webeditor.yml:82]: Nope

**Comment by blindpirate** [comment]
Thanks a lot!

**Pull request merged**